celery/worker/consumer/connection.py

```diff
@@ -18,8 +18,12 @@ def __init__(self, c, **kwargs):
         super().__init__(c, **kwargs)
 
     def start(self, c):
-        c.connection = c.connect()
-        info('Connected to %s', c.connection.as_uri())
+        try:
+            c.connection = c.connect()
```

Comment:
Did you check c.connect() already handles the exception, if yes, this function can't catch the exception.

Reply:
No, but there is a consumer class which has certain number of retries and some generate logs. However, internally consumer tries the connection in above function. With this log we will have exact reason for failure which is not logged by Consumer class.

Reply:
Is there any other places need to add the logs?

Reply:
Yes, added few more logs to track unreleased connections.